src/client/job.rs

```diff
@@ -57,7 +57,7 @@ pub fn job_status(info: &JobInfo) -> Status {
 fn task_status(status: &JobTaskState) -> Status {
     match status {
         JobTaskState::Waiting => Status::Waiting,
-        JobTaskState::Running => Status::Running,
+        JobTaskState::Running(_) => Status::Running,
```

Comment:
@Kobzol Don't we want to propagate it to status too within this PR?

Reply:
I wonder if there is a possible test here where we propagate this to the job_list table, create two tasks, spawn two workers,  see if the tasks are running on different workers

Reply:
Yeah this PR is not too big, we can just directly propagate and display the worker ID in the corresponding tables.